model/src/main/kotlin/io/spine/validation/bound/BoundContext.kt

```diff
@@ -39,6 +41,8 @@ import io.spine.validation.RANGE
  */
 internal open class BoundContext(
```

Comment:
I know this is out of scope, but will it be possible to think of another name for this? :)